pkg/sql/codegen/tensorflow/codegen.go

```diff
@@ -37,6 +37,16 @@ range: [1, Infinity]`, attribute.IntLowerBoundChecker(1, true)},
 	"train.verbose": {attribute.Int, `[default=0]
 Show verbose logs when training.
 possible values: 0, 1`, attribute.IntChoicesChecker([]int{0, 1})},
+	"train.max_steps": {attribute.Int, `[default=0]
```

Comment:
Maybe we need to enhancement the user guide also.

Reply:
`docgen` currently only generate XGBoost documents, will do that in a single PR.